coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4307,6 +4307,34 @@ def max(context, node):
         context.add(values, torch_name=values_name)
         context.add(indices, torch_name=indices_name)
 
+@register_torch_op
+def amax(context, node):
+    inputs = _get_inputs(context, node, expected=[2, 3])
+
+    # mimic functionality from https://pytorch.org/docs/stable/generated/torch.amax.html
+    _input = inputs[0]
+    dim = [inputs[1].val] if type(inputs[1].val) == int else [x for x in inputs[1].val]
+    keepdim = inputs[2] if len(inputs) == 3 else False
+
+    values = mb.reduce_max(x=_input, axes=dim, keep_dims=keepdim)
+    assert len(node.outputs) == 1
+    values_name = node.outputs[0]
+    context.add(values, torch_name=values_name)
+
+@register_torch_op
+def amin(context, node):
```

Comment:
This duplicates a lot of code from `amax` above; is there a pattern used for existing ops that I can follow here to consolidate the impls?

Reply:
Hi @nikalra, thank you so much for the PR!

How about writing a util function, something like
```
def _get_input_amin_amax(context, node):
    inputs = _get_inputs(context, node, expected=[2, 3])
    _input = inputs[0]
    dim = [inputs[1].val] if type(inputs[1].val) == int else [x for x in inputs[1].val]
    keep_dim = inputs[2] if len(inputs) == 3 else False

    return _input, dim, keep_dim


@register_torch_op
def amax(context, node):
    _input, dim, keep_dim = _get_input_amin_amax(context, node)
    context.add(mb.reduce_max(x=_input, axes=dim, keep_dims=keepdim, name=node.name))

@register_torch_op
def amin(context, node):
    _input, dim, keep_dim = _get_input_amin_amax(context, node)
    context.add(mb.reduce_min(x=_input, axes=dim, keep_dims=keepdim, name=node.name))
```

Reply:
Done!